views/beers.hbs

```diff
@@ -0,0 +1,7 @@
+<ul>
+    {{#each beersFromApi}}
+     <li>{{this.name}}</li>
```

Comment:
Could more be rendered here?
The iteration mentions rendering "Display an image, name, description and tagline".

Reply:
I added "{{>beerpartial this}}". Could you please check it?